### PR TITLE
Styled popover component.

### PR DESCRIPTION
**Pull request by kanastasakis26**

### CODE REVIEW

File: src/sass/mmui-styles/mmui-widget/mmui_popover.scss [hunk 1]
```diff
@@ -0,0 +1,175 @@
+/*
+    These styles extend and modify the existing Bootstrap popover styles.
+    The Bootstrap library uses PopoverJS, a third party dependency, to create
+    and display popovers.
+    Popovers are written with HTML, CSS, JS. As previously stated MMUI popover styles
+    are located in this document. The remainder can be found in the Bootstrap and
+    PopoverJS source files.
+ */
```
Comment (xyz8983) [inline src/sass/mmui-styles/mmui-widget/mmui_popover.scss]: Gus is perfect!

File: src/sass/mmui-styles/mmui-widget/mmui_popover.scss [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+  &:hover {
+    color: $mm-ocean;
+  }
+}
+
+.mmui-popover {
+  padding: 16px;
+  border: 0 none;
+  border-radius: 0;
+  filter: drop-shadow(0 3px 6px rgba(0,0,0,0.2));
```
Comment (xyz8983) [inline src/sass/mmui-styles/mmui-widget/mmui_popover.scss]: is this shadow reusable?

File: src/sass/mmui-styles/mmui-widget/mmui_popover.scss [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+      display: none;
+    }
+  }
+}
+
+.bs-popover-top {
+    margin-bottom: $mmui-popover-arrow-height;
+
+    .arrow {
+      bottom: calc((#{$mmui-popover-arrow-height} + #{$popover-border-width}) * -1);
```
Comment (xyz8983) [inline src/sass/mmui-styles/mmui-widget/mmui_popover.scss]: fancy calc function! good to learn

File: src/sass/mmui-styles/mmui-widget/mmui_popover.scss (hunk 2 above)
Comment (besseym) [reply src/sass/mmui-styles/mmui-widget/mmui_popover.scss]: No IE support for filter and only partial support in Edge.

Comment (kanastasakis26) [reply src/sass/mmui-styles/mmui-widget/mmui_popover.scss]: I can go back to using bow-shadow, but I will not be able reproduce the design.

Comment (besseym) [reply src/sass/mmui-styles/mmui-widget/mmui_popover.scss]: I think spend a little more time trying to get box-shadow to work. I can take a look at it as well.

**Pull request merged**